packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.scss

```diff
@@ -3,6 +3,11 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
+.iui-dialog-default.presentation-instance-filter-dialog-content-container {
```

Comment:
Do not use `iui-*` css classes. It is considered an internal API and might change with new `itwinui-react` versions. Second bullet point https://github.com/iTwin/iTwinUI/wiki/Version-conflicts#general-guidelines